crates/evm/core/src/fork/multi.rs

```diff
@@ -85,28 +85,9 @@ impl MultiFork {
     /// Creates a new pair and spawns the `MultiForkHandler` on a background thread.
```

Comment:
this is no longer accurate and will now panic if called outside of tokio

Reply:
hmm, a new thread with current thread runtime is actually a simple solution solution here and makes this usuable outside of tokio

Reply:
https://github.com/foundry-rs/foundry/actions/runs/8123816047/job/22204622755?pr=7288#step:12:222
```
thread '<unnamed>' panicked at /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.36.0/src/time/interval.rs:139:26:
there is no reactor running, must be called from the context of a Tokio 1.x runtime
```